css/css-grid/alignment/grid-gutters-011.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Percentage gap, content-based width/height (via float)</title>
+<link rel="help" href="https://www.w3.org/TR/css-grid-1/#gutters">
+<link rel="help" href="https://www.w3.org/TR/css-align-3/#gaps">
+<link rel="match" href="../../reference/ref-filled-green-100px-square.xht">
+<link rel="author" title="Elika J. Etemad" href="http://fantasai.inkedblade.net/contact">
+<style>
+    #grid {
+        display: grid;
+        grid-gap: 10%;
+        gap: 10%;
+        float: left;
+        width: 100px;
```

Comment:
We cannot have a fixed width here, otherwise the 10% column gap will be resolved to 10px and we'll have a red vertical stripe.